migrations/V190__Add_System_Intake_Software_Acquisition_Info.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TYPE software_acquisition_method AS ENUM ('CONTRACTOR_FURNISHED', 'FED_FURNISHED', 'ELA_OR_INTERNAL', 'NOT_YET_DETERMINED', 'OTHER');
+
+ALTER TABLE system_intakes ADD COLUMN using_software TEXT;
```

Comment:
I'd be tempted to use an `ENUM` value for this instead of `TEXT` so that we can more readily generate types/predict what value this will have. Something like what we did in this other migration?

https://github.com/CMS-Enterprise/easi-app/blob/d58ffa18ecefab49d0b36c925b31c4a18e13f8c8/migrations/V183__Add_GRB_review_members.sql#L15-L20

Reply:
Maybe just a generic enum to represent a "Yes", "No", and "Not Sure" option?

Reply:
could this be a bool?